src/model/element.js

```diff
@@ -116,13 +116,14 @@ export default class Element extends Node {
 	 * @returns {Boolean}
 	 */
 	is( type, name = null ) {
-		const cutType = type.replace( /^model:/, '' );
-
 		if ( !name ) {
```

Comment:
When we talk about micro optimizations `name === null` may be faster.

Reply:
Something doesn't work (maybe `undefined` is passed sometimes).  Tests breaks after suggested change. Looking that this might be to micro optimization to care (looking on above times) I think we shouldn't dig into this more at this time.